### PR TITLE
Add tests for task updating, changing status and clearing completed items 🤓

### PR DESCRIPTION
**Pull request by hsztan**
My partner 🧑‍💻👨🏻‍💻 and I created additional tests ⚙️ for updating, changing status ✅, and clearing the completed items 📝. The basic tests checked the functionality 🛠 of these methods and how they mutated the data 💾 in the DOM, local memory, and local storage. We also checked 👀 if our clearing method would only leave incomplete tasks. 😎

### CODE REVIEW

File: crud.test.js [hunk 1]
```diff
@@ -52,3 +63,136 @@ describe('deleteAndRemoveTask', () => {
     expect(document.querySelector('.todo-item')).toBeFalsy();
   });
 });
+
```
Comment (eri8-9) [inline crud.test.js:66]: - [ ] 8 of your tests failed. It'd be great if all of them can be pass the test to continue. 😉

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️

- [x] In your readme file all the members should add their social media links.
- [x] In your readme file contain the instructions necessary to run the tests with jest.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: crud.test.js (hunk 1 above)
Comment (AKeeganDev) [reply crud.test.js:66]: @eri8-9 

Thanks for the review! Can you provide us a screenshot of the failed tests? We're green across the board here so we're not sure what you might be experiencing. Can you provide some more context so we can troubleshoot? This is how ours looks:

image

Comment (AKeeganDev) [reply crud.test.js:66]: @eri8-9 

Thanks this is helpful and I see the issue you're experiencing. It looks like the version of node that you're running is not recognizing the latest syntax which is why it is failing on the tests using .at(). Our version  of Node (after calling "node --version" in the terminal) is 16.14.2. Can you update your node and try again? I believe that is the issue. Some documentation on .at() can be found here if you need to validate that it is valid syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**